ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/PrettyValidationErrors.scala

```diff
@@ -42,13 +37,7 @@ object PrettyValidationErrors {
       case MissingParameters(params, _) =>
         node(s"Node parameters not filled", s"Please fill missing node parameters: : ${params.mkString(", ")}")
 
-      // Parameter validation error cases
-      case ErrorValidationParameter(validator, paramName, _) => validator match {
-        case MandatoryParameterValidator =>
-          validation(validator, s"No expression found for mandatory parameter", s"Please fill expression for this parameter", fieldName = Some(paramName))
-        case NotBlankParameterValidator =>
-          validation(validator, s"Blank expression for not blank parameter", s"Please fill expression for this parameter", fieldName = Some(paramName))
-      }
+      case pve: ParameterValidationError => handleParameterValidationError(pve)
```

Comment:
Please do not use 3 letter abbreviations, use some meanigfull name instead :)

Reply:
Man it's change from old PR (https://github.com/TouK/nussknacker/pull/795/files#diff-ea4056b249d590b498c1de2368661b86R40) which you've already have approved ;)

Reply:
The comment is still up to date, but of course not crucial